composites/Plugin/Shared/components/HelpText.js

```diff
@@ -1,30 +1,42 @@
 /* External dependencies */
-import React from "react";
+import React, { PureComponent } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
+/* Internal dependencies */
+import colors from "../../../../style-guide/colors";
+
 const YoastHelpText = styled.p`
-	font-size: 13px;
+	color: ${ props => props.color };
+	font-size: ${ props => props.fontSize };
+	margin-top: 0;
+	
+	a {
+        color: ${ colors.$palette_blue_medium };
+    }
 `;
 
 /**
  * Returns the HelpText component.
  *
- * @param {Object} props Component props.
+ * @param {Object} props          Component props.
+ * @param {string} props.text     The help text.
+ * @param {string} props.color    The CSS color of the paragraph text.
+ * @param {string} props.fontSize The CSS font-size of the paragraph text.
  *
  * @returns {ReactElement} HelpText component.
  */
-export default class HelpText extends React.Component {
+export default class HelpText extends PureComponent {
 	/**
 	 * Renders a help text component.
 	 *
 	 * @returns {ReactElement} The rendered help text component.
 	 */
 	render() {
-		const { text } = this.props;
+		const { text, color, fontSize } = this.props;
 
 		return (
-			<YoastHelpText>
+			<YoastHelpText color={ color } fontSize={ fontSize } >
```

Comment:
See renaming the `fontSize` prop

Reply:
Oh well, just realised this applies also to the `color` prop, which is rendered in the markup as HTML attribute:
`<p class="HelpText__YoastHelpText-bnJWjb gHxXTM" color="#888" font-size="0.8em">`

So also the `color` prop should be renamed, e.g. to `textColor`.